tensorflow/python/keras/backend.py

```diff
@@ -1464,30 +1465,37 @@ def batch_dot(x, y, axes=None):
   """Batchwise dot product.
 
   `batch_dot` is used to compute dot product of `x` and `y` when
-  `x` and `y` are data in batch, i.e. in a shape of
+  `x` and `y` are data in batches, i.e. in a shape of
   `(batch_size, :)`.
   `batch_dot` results in a tensor or variable with less dimensions
   than the input. If the number of dimensions is reduced to 1,
   we use `expand_dims` to make sure that ndim is at least 2.
 
-  Arguments:
+  # Arguments
       x: Keras tensor or variable with `ndim >= 2`.
       y: Keras tensor or variable with `ndim >= 2`.
-      axes: list of (or single) int with target dimensions.
-          The lengths of `axes[0]` and `axes[1]` should be the same.
+      axes: int or tupe(int, int). Target dimensions to be reduced.
 
-  Returns:
+  # Returns
```

Comment:
Please fix incorrect docstring formatting (see other docstrings in the TF codebase)

Reply:
Ping @farizrahman4u Can you fix this so we can merge the PR? I'll leave it up to you if you want to add unit test for error conditions. It's not critical, but nice to have.